### PR TITLE
Copy changes to proposal page

### LINKED ISSUE #626: Additional copy for top of call for proposals page
During yesterdays call it was highlighted that we need a bit more context about the type of sessions we are looking for. I want to de-emphasizes slide presentations and lecture but ensure people understand they can submit a wide variety of sessions such as art pieces and hacks.
What do people think about this as a copy edit @mmmavis @KevZawacki @hannahkane to the top of this page, first line: https://mozillafestival.org/proposals

MozFest sessions cover a wide range of topics, but share two important qualities: they are interactive and inclusive. We welcome workshops, exhibits and often shy away from lectures and slide presentations.

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #626

### CODE REVIEW

**Comment by mmmavis** [comment]
@Saallen @edrushka @TheoChevalier can you help QA and see the following pages have the right copy?

- http://mozillafestival-org-sta-pr-639.herokuapp.com/proposals
- http://mozillafestival-org-sta-pr-639.herokuapp.com/proposals/espanol
- http://mozillafestival-org-sta-pr-639.herokuapp.com/proposals/francais
- http://mozillafestival-org-sta-pr-639.herokuapp.com/proposals/deutsch

**Comment by edrushka** [comment]
EN, FR and ES look good to me. 

I can't read enough German to be sure, but it appears the additional text is in the right spot.

**Comment by edrushka** [comment]
@mmmavis I did notice that in https://github.com/mozilla/mozillafestival.org/pull/639#issuecomment-310535284 @Saallen specifically said it needs to be in italics. You might get her to sign off on this.

**Comment by TheoChevalier** [comment]
@edrushka I’ll let Sarah clarify if needed, but I think italics were just here for us to notice the new part from the existing part.

In Spanish I think there’s a tiny typo, `informaciónn` should be `información`, ccing @avelper for confirming.
French and German are looking good

**Comment by Saallen** [comment]
Hello @mmmavis @TheoChevalier @edrushka 
Those italics were just to highlighted the requested changes,.
Thanks for acting so fast with the edits and translations all :champagne:

**Comment by avelper** [comment]
@TheoChevalier You are right, it should be información. If you tell me where, I can change it.

Thanks!

**Comment by TheoChevalier** [comment]
Right on, thanks! Just noticed you’ve updated the string in Pontoon

File: pages/proposals/language/spanish.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
...
         "Las sesiones pueden albergar de 3 a 25 participantes, de diferentes edades y con distintas habilidades. Todos los participantes del festival pueden acudir a tu sesión.",
-        "Cualquiera puede enviar una propuesta para MozFest 2017. Cuando la hayas enviado, tu sesión entrará a formar parte de un proceso abierto en <a href=\"https://github.com/MozillaFoundation/mozfest-program-2017\" target=\"_blank\">GitHub</a>",
+        "Cualquiera puede enviar una propuesta para MozFest 2017. Cuando la hayas enviado, tu sesión entrará a formar parte de un proceso abierto en <a href=\"https://github.com/MozillaFoundation/mozfest-program-2017\" target=\"_blank\">GitHub</a>. Toda la información que envíes por aquí, excepto tu identificador de Twitter y correo electrónico, se publicará en GitHub. Mozilla puede ponerse en contacto contigo a través del correo en relación a la informaciónn que enviaste.",
```
Comment (TheoChevalier) [inline pages/proposals/language/spanish.json]: Commenting here so that we don’t forget.
`a la informaciónn` → `a la información`

**Pull request merged**